syntax = "proto3";

package syft.core.node.domain.service;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message PublishScalarsAction {
  syft.core.common.UID id_at_location = 1;
  syft.core.common.UID msg_id = 2;
  syft.core.io.Address address = 3;
  repeated syft.core.common.UID publish_ids_at_location = 4;
  float sigma = 5;
}
